tRecorderApi/api/tests/tests_get_projects.py

```diff
@@ -13,52 +11,92 @@
+                                           anthology=self.anthology, language=self.lang,
+                                           book=self.book)
+        self.chap = Chapter.objects.create(number=1, checked_level=1, published=False, project=self.proj)
+        self.chap2 = Chapter.objects.create(number=2, checked_level=2, published=False, project=self.proj)
+        self.chap3 = Chapter.objects.create(number=3, checked_level=3, published=False, project=self.proj)
+        self.chap4 = Chapter.objects.create(number=4, checked_level=0, published=False, project=self.proj)
+
         self.chunk = Chunk.objects.create(startv=0, endv=3, chapter=self.chap)
-        self.user = User.objects.create(name='testy', agreed=True, picture='mypic.jpg')
-        self.take = Take.objects.create(location=location_wav, is_publish=True,
-                                   duration=0, markers="{\"test\" : \"true\"}", rating=2, chunk=self.chunk, user=self.user)
+        self.chunk2 = Chunk.objects.create(startv=0, endv=3, chapter=self.chap2)
+        self.chunk3 = Chunk.objects.create(startv=0, endv=3, chapter=self.chap3)
+        self.chunk4 = Chunk.objects.create(startv=0, endv=3, chapter=self.chap4)
+        self.chunk5 = Chunk.objects.create(startv=0, endv=3, chapter=self.chap4)
+        self.chunk6 = Chunk.objects.create(startv=0, endv=3, chapter=self.chap4)
+
+        self.take = Take.objects.create(location=location_wav, published=True,
+                                        duration=0, markers="{\"test\" : \"true\"}", rating=2, chunk=self.chunk
+                                        )
+        self.take = Take.objects.create(location=location_wav, published=True,
+                                        duration=0, markers="{\"test\" : \"true\"}", rating=2, chunk=self.chunk2
+                                        )
+        self.take = Take.objects.create(location=location_wav, published=True,
+                                        duration=0, markers="{\"test\" : \"true\"}", rating=2, chunk=self.chunk3
+                                        )
+        self.take = Take.objects.create(location=location_wav, published=True,
+                                        duration=0, markers="{\"test\" : \"true\"}", rating=2, chunk=self.chunk4
+                                        )
+
         self.project_takes_data = {"language": "en-x-demo2", "version": "ulb", "book": "mrk", "chapter": 1}
 
-    def test_that_we_can_get_projects(self):
-        """Testing that submitting a POST request through key search returns a JSON object"""
+    def test_get_minimum_checked_level(self):
+        min_check_level = Chapter.objects.all().values_list('checked_level') \
+            .order_by('checked_level')[0][0]
+        self.assertEqual(min_check_level, 0)
+
+    def test_get_total_chunks(self):
+        chunks_done = Chapter.objects.all().values_list('chunk').count()
+        self.assertEqual(chunks_done, 6)
+
+    @staticmethod
+    def test_build_dict():
```

Comment:
@antonlopez this will get run as a test because of the name but appears to be a helper method